api/services/reporting/report_generator.py

```
@@ -0,0 +1,207 @@
+            bytes: PDF file as bytes
+
+        Raises:
+            Exception: If PDF generation fails
+        """
+        try:
+            logger.info("Starting PDF generation for verification report")
+
+            # Prepare context data
+            context = {
+                'resultado': resultado,
+                'timestamp': datetime.now().strftime("%d/%m/%Y às %H:%M:%S"),
```

Ruff (0.14.5)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)